excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImpl.java

```diff
@@ -200,6 +204,9 @@ public DefinitionFileUploadMetadata importFormDefinitions(InputStream inputStrea
 
         metadata.setUserId(userDetails.getEmail());
 
+        logger.info("Clearing the case definition cache");
```

Comment:
This sounds like a debug log, rather than an info one.

Reply:
just that all preceding logger statements are info and definition imports are not so frequent. We could live with an info here, wont clutter the logs.

Reply:
makes complete sense..